plugins/org.python.pydev.ast/src/org/python/pydev/ast/interpreter_managers/InterpreterInfo.java

```diff
@@ -1897,7 +1899,24 @@ public void setStringSubstitutionVariables(Properties stringSubstitutionOriginal
 
     @Override
     public Properties getStringSubstitutionVariables() {
-        return this.stringSubstitutionVariables;
+        return getStringSubstitutionVariables(false);
+    }
+
+    @Override
+    public Properties getStringSubstitutionVariables(boolean addEnvironmentVaiables) {
+        Properties ret = new Properties();
+        ret.putAll(this.stringSubstitutionVariables);
+        if (addEnvironmentVaiables) {
+            Map<String, String> formattedEnv = new Hashtable<String, String>();
+            System.getenv().forEach(new BiConsumer<String, String>() {
```

Comment:
If you're going to use a `forEach` instead of a `for` loop, use a lambda function here. i.e.: https://www.w3schools.com/java/java_lambda.asp